metrics/share-of-voice.md

```diff
@@ -0,0 +1,56 @@
+# Share of Voice
+
+Can we communicate the value of our community as a comparison to something of value by our supporting business?
+
+## Description
+
+Ratio of conversation occurring by or about our community in comparison to some baseline. Also described as: Visibility, chatter, social media presence, top-of-funnel impact.
+
+## Objectives
+
+The metric supports the perception of continued value from community work in order to continue to secure funding. It highlights the value driven by community work by comparing it to something already valued by the organization. 
+
+## Implementation
+
+### Prerequisites
+
+Share of voice only works if you can agree to its implied assumptions ahead of time:
+
+- The more conversation about you, the better (which requires accepting the adage ‘there is no such thing as bad publicity’).
+- You have a public location worth measuring (public content like blogs or reviews, Twitter conversation, conference talks, StackOverflow tags).
+- There is a competitor that comes to mind, whether its internal competition where you compare community SOV to PR or AR efforts or external, where you focus on the top 3 companies in a given space (ex. comparing how N projects from X, Y, or Z companies compete for conversation around Data Science).
```

Comment:
SOV = Share of Voice?
PR = Public Relation?
AR = ???

```suggestion
- The community has a competitor, either in the public or internal to an organization. For the latter case, Share of Voice can be compared to public relations or ?AR? efforts of internal competitors. It is suggested to focus on the top 3 competitors in a given space (e.g., comparing how open source communities from X, Y, or Z companies compete for conversation around Data Science).
```

project --> community, to stay consistent 
Rephrased the text a bit.